konfera/event/views.py

```diff
@@ -123,3 +124,23 @@ def get_context_data(self, **kwargs):
         set_event_ga_to_context(event, context)
 
         return context
+
+
+def event_public_tickets(request, slug):
+    context = dict()
+
+    event = get_object_or_404(Event.objects.published(), slug=slug)
+    context['event'] = event
+    available_tickets = event.tickettype_set.filter(accessibility=PUBLIC)
```

Comment:
Improve to filter also by ticket dates, so we do not dispaly expired tickets and also those that will by available in future.